app/src/main/java/com/texthip/thip/ui/mypage/viewmodel/SavedBookViewModel.kt

```
@@ -82,10 +83,14 @@ class SavedBookViewModel : ViewModel() {
     }
-
     fun toggleBookmark(id: Int) {
-        _bookList.value = _bookList.value.map {
+        _books.value = _books.value.map {
             if (it.id == id) it.copy(isSaved = !it.isSaved) else it
         }
     }
+}
+
+class EmptySavedBookViewModel : SavedBookViewModel() {
+    override fun loadMockBooks() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)